### PR TITLE
Add email guidelines to handbook

### PR DESCRIPTION
**Pull request by iskerrett**
Added some guidelines on sending email from FlowFuse.

### CODE REVIEW

File: src/handbook/marketing/email.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+### FlowFuse Cloud Trial email
+
+The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud. 
+
+These emails will be sent from HubSpot using data based on the user interaction with FlowFuse Cloud. A system has been put in place to query the FF Cloud database to create events that are added to HubSpot. HubSpot workflows will trigger the emails based on these events.
+
+The expectation is that the marketing team will continuously test different types of email, and email content to improve the conversion rate of the trial period.
+
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:32]: Who does anyone propose changes to these emails?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/handbook/marketing/email.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+meta:
```
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/handbook/marketing/email.md:2]: Sure, how do I do that?

File: src/handbook/marketing/email.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The sales team, customer success and support team will be required to send email to communicate with specific potential customers and customers. These types of email are typically between the FlowFuse employee and an individual customer to accomplish a task or transaction. For instance, handling a support request or further a sales process.
+
+These types of email should be stored in HubSpot using the HubSpot interface or the HubSpot Chrome plugin. The goal should be that all transactional email is associated with the contact in Hubspot to have an history of the interaction.
+
+### Community email 
+
+The marketing team is responsible for sending email to all community members, potential customers and customers that provide updates on FlowFuse and events organized by FlowFuse. Examples of these email include monthly newsletter, webinar promotion, product release, major company announcements, etc.
+
+These types of emails should be sent via HubSpot.
```
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/handbook/marketing/email.md:23]: The marketing team is responsible so only the marketing team should be sending these emails.  People can certainly request an email to be sent but it needs to be sent by the marketing team.

File: src/handbook/marketing/email.md (hunk 1 above)
Comment (iskerrett) [reply src/handbook/marketing/email.md:31]: Anyone can open a issue in github to request a change. I can add this to the document if you want.

File: src/handbook/marketing/email.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/handbook/marketing/email.md:2]: Add a line to `src/handbook/marketing/index.md`

File: src/handbook/marketing/email.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply src/handbook/marketing/email.md:23]: Can you update the handbook to say so?

File: src/handbook/marketing/email.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/handbook/marketing/email.md:2]: done

File: src/handbook/marketing/email.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (iskerrett) [reply src/handbook/marketing/email.md]: Done

File: src/handbook/marketing/email.md (hunk 1 above)
Comment (iskerrett) [reply src/handbook/marketing/email.md:33]: Done

File: src/handbook/marketing/email.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The sales team, customer success, and support team will be required to send email to communicate with specific potential customers and customers. These types of email are typically between the FlowFuse employee and an individual customer to accomplish a task or transaction. For instance, handling a support request or further a sales process.
+
+These types of email should be stored in HubSpot using the HubSpot interface or the HubSpot Chrome plugin. The goal should be that all transactional email is associated with the contact in Hubspot to have an history of the interaction.
+
+### Community email 
+
+The marketing team is responsible for sending email to all community members, potential customers and customers that provide updates on FlowFuse and events organized by FlowFuse. Examples of these email include monthly newsletter, webinar promotion, product release, major company announcements, etc.
+
+These types of emails should be sent via HubSpot. To ensure consistency and continuity, the marketing team should be the only team sending community emails via HubSpot. Anyone can request an email to be sent by opening a github issue on the website.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:24]:
```suggestion
These types of emails should be sent via HubSpot. To ensure consistency and continuity, the marketing team should be the only team sending community emails via HubSpot. Anyone can request an email to be sent by opening a [GitHub issue on the website repository](https://github.com/flowforge/website/issues).
```

File: src/handbook/marketing/email.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### FlowFuse Cloud Trial email
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:26]:
```suggestion
### FlowFuse Cloud User emails
```

File: src/handbook/marketing/email.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:28]:
```suggestion
The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful usage of FlowFuse Cloud and users growth in adoption.
```

File: src/handbook/marketing/email.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+### FlowFuse Cloud Trial email
+
+The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud. 
+
+These emails will be sent from HubSpot using data based on the user interaction with FlowFuse Cloud. A system has been put in place to query the FF Cloud database to create events that are added to HubSpot. HubSpot workflows will trigger the emails based on these events.
+
+The expectation is that the marketing team will continuously test different types of email, and email content to improve the conversion rate of the trial period.
+
+FlowFuse Cloud Trial emails should only be sent by the marketing team via HubSpot. Anyone can suggestion changes by opening a github issue.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:34]:
```suggestion
FlowFuse Cloud Trial emails should only be sent by the marketing team via HubSpot. Requesting changes to these emails is done by opening a
[GitHub issue on the website repository](https://github.com/flowforge/website/issues).
```

File: src/handbook/marketing/email.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+These types of emails should be sent via HubSpot. To ensure consistency and continuity, the marketing team should be the only team sending community emails via HubSpot. Anyone can request an email to be sent by opening a github issue on the website.
+
+### FlowFuse Cloud Trial email
+
+The marketing team, in collaboration with the customer success team, is responsible for defining and creating an email campaign to encourage the successful trial of FlowFuse Cloud. 
+
+These emails will be sent from HubSpot using data based on the user interaction with FlowFuse Cloud. A system has been put in place to query the FF Cloud database to create events that are added to HubSpot. HubSpot workflows will trigger the emails based on these events.
+
+The expectation is that the marketing team will continuously test different types of email, and email content to improve the conversion rate of the trial period.
```
Comment (ZJvandeWeg) [inline src/handbook/marketing/email.md:32]:
```suggestion
The expectation is that the marketing and customer success team will continuously test different types of email, and email content to improve the conversion rate of the trial period and adoption beyond of FlowFuse Cloud.
```

Comment (iskerrett) [reply src/handbook/marketing/email.md:32]: Not sure what you mean by 'and adopt beyond of'?

File: src/handbook/marketing/email.md (hunk 6 above)
Comment (iskerrett) [reply src/handbook/marketing/email.md:28]: how about we say 'successful usage and adoption of FlowFuse Cloud.'

File: src/handbook/marketing/email.md (hunk 8 above)
Comment (ZJvandeWeg) [reply src/handbook/marketing/email.md:32]: For example; growing from starter package to the Team tier. But also other utilisation topics, like growing starter users to adopt devops pipelines.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**